dump/dumper.go

```diff
@@ -202,10 +202,25 @@ func (d *Dumper) Dump(w io.Writer) error {
 	if strings.Contains(d.Addr, "/") {
 		args = append(args, fmt.Sprintf("--socket=%s", d.Addr))
 	} else {
-		seps := strings.SplitN(d.Addr, ":", 2)
-		args = append(args, fmt.Sprintf("--host=%s", seps[0]))
-		if len(seps) > 1 {
-			args = append(args, fmt.Sprintf("--port=%s", seps[1]))
+		ipv6 := strings.Count(d.Addr, ":") > 1
```

Comment:
this part can also use `net.SplitHostPort`, just like canal/canal.go

Reply:
done